portfolio/src/main/java/com/google/sps/servlets/RestaurantDataServlet.java

```diff
@@ -44,9 +43,11 @@ public void init() {
     readRestaurants(3);
   }
 
-  /** Reads restaurant data from CSV file.
-      Dataset can be found here: https://www.kaggle.com/jackywang529/michelin-restaurants */
-  private void readRestaurants(int star){
+  /**
+     Reads restaurant data from CSV file.
+      Dataset can be found here: https://www.kaggle.com/jackywang529/michelin-restaurants
+   */
+  private void readRestaurants(int star) {
     String path = "/WEB-INF/" + NUMBERS.get(star) + "-stars-michelin-restaurants.csv";
     Scanner scanner = new Scanner(getServletContext().getResourceAsStream(path));
```

Comment:
In separate pull request: you could use try-with-resources: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html